R/Functions_L2_Colony.R

```diff
@@ -1304,12 +1315,23 @@ swarm <- function(x, p = NULL, year = NULL, nVirginQueens = NULL, simParamBee =
     #       https://github.com/HighlanderLab/SIMplyBee/issues/160
     tmp <- pullWorkers(x = x, nInd = nWorkersSwarm)
     currentLocation <- getLocation(x)
+    if (sampleLocation) {
+      if (length(sdLocation) != 2) {
```

Comment:
What if it was one number - the radius?

Reply:
Hmm, at the moment we operate on (x,y) coordinates, and to sample a new (x,y) we need (dx,dy) where dx~N(0,sigma_{dx}^2) and dy~N(0,sigma_{dy}^2) so we need sigmas (SD) as input. This means that I did not need to implement the radius stuff at all;)

Having said this, someone might have idea about the radius but not the sigmas, which is what you are hinting at, correct? Maybe we provide a function to convert radius to sigmas? I guess also if we have a radius, then the two sigmas are equal, right? Is that really what you are asking? If so, can you copy this is a new spatial issue and we take it from there?

Reply:
Yes, I was referring to sigmas being equal if we talk about a radius. Now discussed in #9